docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.7-slim
+
+RUN apt-get update -q \
+    && apt-get install --no-install-recommends -y \
+        ffmpeg \
+        gcc \
+        git \
+        libcairo2-dev \
+        libffi-dev \
+        libpango1.0-dev \
```

Comment:
Not required.
```suggestion
```

Reply:
True! I am not sure how this slipped in.